### DISCUSSION TITLE
Unable to receive verification code via sms

### DISCUSSION THREAD

**Discussion by Mich45** [vercel/community#714] [Help] [answered]
Hi everyone,

I have been unable to login into vercel with my GitHub account. I got stuck at the verification stage. I have tried using another phone number to receive the verification code via SMS but I still can't get it. 
Your help will be greatly appreciated. Thanks

![vercel_LI](https://user-images.githubusercontent.com/46695719/178169232-ec89b4c3-1767-4a74-a349-aa2b107747dd.jpg)

**Comment by amyegan** [comment, marked as answer]
Hi @Mich45. I want to keep your private info safe and off of this public discussion. Please send us an email at registration@vercel.com with the email address or username used to sign up, and the team will help get your account ready for use!

**Reply by Mich45** [reply]
Thanks for the prompt response! I'll do that right away!

**Reply by Mich45** [reply]
Thank you so much @amyegan!
I think it's fixed now. I can now login to my account!

**Reply by Alie-Kiaalhosseini** [reply]
hi @amyegan. I have the same issue and I've just sent email to registration@vercel.com.hope it will be fixed.

**Reply by a7xr** [reply]
Hi @amyegan, I have the same issue as well,
Going to send an email to registration@vercel.com

Hope you can help

**Reply by amyegan** [reply]
Sorry you're having trouble with login. We never want to expose your account info here, so email is the definitely best approach 🙇
